Automattic/vip-go-mu-plugins
# Use full file path for file service caching

## Description

The current file service API cache uses the file base name as the cache key but this causes problem when a file is uploaded with the same name but on a different path. Functions like `file_exist` will return `true` even though the file is on a different path.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
1. Upload changes to a sandboxed site
1. `wp --allow-root shell`
1. Upload a new file: `file_put_contents( 'vip://wp-content/uploads/path1/test.txt', 'test' );`
1. `file_exist( 'vip://wp-content/uploads/path2/test.txt' );`
1. Verify that `file_exist` call returns `false`.
